src/outlet/interfaces.ts

```diff
@@ -1,4 +1,5 @@
 import { TokenType } from './ticketStorage'
+import { AbiFieldTypes } from '@tokenscript/attestation/dist/eas/EasTicketAttestation'
```

Comment:
What are your thoughts for us to use a naming convention here. In case we need to introduce various AbiFieldTypes? e.g. for tokens beyond ticketAttestation. However this is ok for now, we can iterate changes here etc if preferred.

Reply:
@nicktaras AbiFieldTypes are always the same, unless ethereum adds new ones but I doubt it.